back/app/policies/custom_page_file_policy.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
```

Comment:
Again, not entirely sure I need to do this at this point, but I've not worked with our files stuff before, so I guessed at yes.

Reply:
You use `Scope`, but not the entire file. So, I would not add the rest at the beginning.

Reply:
> I would not add the rest at the beginning.

@alexander-cit Which part would you not add?

Reply:
Note: I based this on [initiative_file_policy.rb](https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/app/policies/initiative_file_policy.rb)

Reply:
I would leave only the `Scope` class. And probably we don't need methods `create?`, `show?`, `update?`, `destroy?` and others, so maybe we could remove them.

Reply:
I think we agreed that since `StaticPageFilePolicy` looks identical to this (and other such policies), we should stick to the convention for now, and leave this as it is now in this PR, and consider changing later if needed/desired.

Reply:
I believe duplicating code by convention is fine, but maybe I would avoid copying unused code. It will definitely confuse in the future.

Reply:
@alexander-cit I understand the wish to avoid redundant code. Is there some specific redundant code you think I should remove, given that this file is an almost exact copy of `CustomPageFilePolicy` or should we wait and see how the current iteration progresses and then revisit the issue of redundant code from this PR later?

Reply:
> should we wait and see how the current iteration progresses and then revisit the issue of redundant code from this PR later?

It would be the best option, but the practice shows us that we forget about the code as soon as we merge it :smile: 

So, I rather propose to keep it minimal and extend it when needed.

Reply:
@alexander-cit So... remove the `create?`, `show?`, `update?`, `destroy?` and keep the...
```
class CustomPageFilePolicy < ApplicationPolicy
  class Scope
````
?

Reply:
And I meant these methods

create?, show?, update?, destroy?

Reply:
Yes.